src/modules/ImageCreation/reducers/imageCreationInitializationReducer.ts

```diff
@@ -59,15 +59,15 @@ export default handleActions({
 	[types.SELECT_ENTITY]: (state: any , {payload: {entity, type, isSelected}}) => {
 		let isFounded = false;
 		let nextIndex = parseInt(state.currentInitializationSlide, 10);
-		const entitesFilters = {...state.entitesFilters};
-		const filtersTypes = Object.keys(entitesFilters);
+		const entitiesFilters = {...state.entitiesFilters};
```

Comment:
what the mess inside that function? talk with Eugene, he helps you to find a better solution for this.